doc/guides/maintaining-npm.md

```diff
@@ -0,0 +1,66 @@
+# Maintaining npm in Node.js
+
+## Step 1: Clone npm
+
+```console
+$ git clone https://github.com/npm/npm.git
+$ cd npm
+```
+
+or if you already have npm cloned make sure the repo is up to date
+
+```console
+$ git remote update -p
+$ git reset --hard origin master
+```
+
+## Step 2: Build release
+
+```console
+$ git checkout vX.Y.Z
+$ make release
```

Comment:
I would add a note to run `npm dist-tag ls npm` and make sure this is the `latest` **dist-tag**. `latest` on git is usually released as `next` when it's time to downstream.

Reply:
fixed below